packages/cli/src/util/config/files.ts

```diff
@@ -6,11 +6,11 @@ import { fileNameSymbol } from '@vercel/client';
 import getGlobalPathConfig from './global-path';
 import getLocalPathConfig from './local-path';
 import { NowError } from '../now-error';
-import error from '../output/error';
 import highlight from '../output/highlight';
 import { VercelConfig } from '../dev/types';
 import { AuthConfig, GlobalConfig } from '@vercel-internals/types';
 import { isErrnoException, isError } from '@vercel/error-utils';
+import { type Output } from '../output';
```

Comment:
```suggestion
import type { Output } from '../output';
```

If all imports are type-only imports, then it's preferable to use `import type`. See: https://x.com/mattpocockuk/status/1795040643963555859